supply_energy.md

```diff
@@ -0,0 +1,291 @@
+
+##### Hydropower
+
+Hydropower is the simplest of all energy forms in GCAM; the quantity of hydropower produced in each region and year is exogenous, prescribed as a "fixedOutput". Hydropower costs are not estimated, and the technology does not contribute to the modeled electricity price in each region. The quantities in future years are generally consistent with the long-term "economic" potential identified by the International Hydropower Association [(IHA 2000)](supply_energy.html#iha2000).
+
+##### Biomass
+
+While most of the effort in modeling biomass supply is in the [agriculture and land use component](land.html), there is a renewable resource represented in the energy system, that generally refers to municipal and industrial wastes that can be used for energy purposes. The supply curves use the same functional form as printed in the Wind section above, and the specific quantities are documented in [Gregg and Smith (2010)](supply_energy.html#gregg2010). Unlike other resources, the waste biomass supply curve is assumed to grow with GDP, as prescribed by the exogenous supply elasticity of GDP, or "gdpSupplyElast".
+
+##### Traditional biomass
+
+Traditional biomass in GCAM is defined as the IEA's "primary solid biomass" product consumed by the residential sector, in selected regions where it is considered to be an important part of the energy system. The largest consumers of traditional biomass in 2010 were China, India, and Western Africa. The specific energy goods involved include firewood, agricultural residues, animal dung, and others; no effort is made to disaggregate the category into these consistuent parts, or to link the production volumes with the agriculture and land use module.
+
+
+### Energy Transformation
+
+Broadly, the energy transformation sectors in GCAM consist of all supplysectors between the [primary energy resources](supply_energy.html#resources) and the [final energy demands](demand_energy.html). The main energy transformation sectors highlighted in this documentation are [electricity](supply_energy.html#electricity), [refining](supply_energy.html#refining), [gas processing](supply_energy.html#en_gas_processing), [hydrogen production](supply_energy.html#hydrogen), and [district services](supply_energy.html#district-services). For more details, see [energy transformation](details_energy.html#energy-transformation)
```

Comment:
- "For more details, see [energy transformation](details_energy.html#energy-transformation)" I think this is another self-reference that should be deleted.  If it remains, the sentence needs a period.